### DISCUSSION TITLE
Is there a way to ignore a sheet from evaluation?

### DISCUSSION THREAD

**Discussion by lumatax** [jxlsteam/jxls#303] [Q&A] [answered]
Hi,

I have a multisheet workbook template. There are certain sheets that I do not want the WorkbookEvaluator to evaluate. Is it possible to ignore sheets? Asking because when it gets evaluated I get an error 

`org.apache.poi.ss.formula.WorkbookEvaluator -- Evaluated Cover Sheet!Y13 to org.apache.poi.ss.formula.eval.ErrorEval [~CIRCULAR~REF~]`

**Comment by SoltauFintel** [comment, marked as answer]
Jxls processes only sheets that have a jx:area command.

Please supply a Github repo with an example. (actual result / expectation)
